src/core/stompest/protocol/commands.py

```diff
@@ -120,7 +121,10 @@ def subscribe(destination, headers, receipt=None, version=None):
         if (version != StompSpec.VERSION_1_0):
             raise
     token = (StompSpec.DESTINATION_HEADER, destination) if (subscription is None) else (StompSpec.ID_HEADER, subscription)
-    return frame, tuple(map(unicode, token))
```

Comment:
Please factor out the distinction of cases s.t. no redundant code remains, e.g. using a ternary `... if ... else ...`).